VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
@vladon @shiftkey's point is that we _already_ have `*.opendb` in this file. So the addition of `*.VC.VC.opendb` does nothing. Either we remove `*.VC.VC.opendb` or we remove `*.opendb`.

@anurse was the one who added `.opendb` in a393ef8de999946cbb0210c24bb8ba39ffb511c0 so I'm not sure what the best call here is.